cli/cli.go

```diff
@@ -207,14 +207,25 @@ func Handle(cliArgs []string) (string, *Options, error) {
 	return cmd, opts, err
 }
 
+// convertSliceArgs splits up comma delimited flags into a slice. We do this because slice argument
+// environment variables in kingpin are newline delimited for some odd reason
+// See https://github.com/alecthomas/kingpin/issues/257
 func convertSliceArgs(opts *Options) {
-	if len(opts.RedisPubSub.Channels) != 0 {
```

Comment:
What's the significance in testing for `!= 0` vs `> 0`?

Reply:
Just made more sense to me since it will never be negative

Reply:
Thinking about this more, I changed it to `== 1` since that would be the correct scenario